Firmata.h

```diff
@@ -92,10 +91,10 @@ class FirmataClass
     void write(byte c);
 
     /* attach & detach callback functions to messages */
-    void attach(uint8_t command, ::callbackFunction newFunction);
-    void attach(uint8_t command, ::systemCallbackFunction newFunction);
-    void attach(uint8_t command, ::stringCallbackFunction newFunction);
-    void attach(uint8_t command, ::sysexCallbackFunction newFunction);
+    void attach(uint8_t command, callbackFunction newFunction);
```

Comment:
`::` points to the global namespace. I tucked them into `FirmataClass` as they are related to its implementation and names like `systemCallbackFunction` can be ambiguous.

I left the globals in place so they can be referenced by existing sketches, but can be easily removed in the future (v3.0).